pyproject.toml

```diff
@@ -35,6 +35,8 @@ pre-commit-hooks = "^4.4.0"
 yamllint = "^1.29.0"
 black = "^23.1.0"
 isort = "^5.12.0"
+autopep8 = "^2.0.1"
+flake8 = "^6.0.0"
```

Comment:
Why are these required? Perhaps remove them from the dependencies?

Reply:
For autoformatting and autolinting this was my favourites and always in my vscode settings